addons/bestja_page_fixtures_ucw/templates/pages/home_page.xml

```diff
@@ -39,8 +39,8 @@
                                                 </div>
                                                 <div class="carousel_title">
                                                     <h1>
-                                                        Use your professional skills <br />
-                                                        to make a difference2
+                                                        <a class="color_white" href="#choose_what_to_do">Use your professional skills <br />
```

Comment:
In this case many names of the classes will have to be changed. I am working on it.